utils/validators-info.js

```diff
@@ -8,15 +8,21 @@ async function validatorsInfo(near, blockNumberOrHash) {
         blockNumberOrHash = Number(blockNumberOrHash);
     }
     const genesisConfig = await near.connection.provider.sendJsonRpc('EXPERIMENTAL_genesis_config', {});
+    const protocolConfig = await near.connection.provider.sendJsonRpc('EXPERIMENTAL_protocol_config', { "finality": "final" });
     const result = await near.connection.provider.sendJsonRpc('validators', [blockNumberOrHash]);
     result.genesisConfig = genesisConfig;
+    result.protocolConfig = protocolConfig;
     result.numSeats = genesisConfig.num_block_producer_seats + genesisConfig.avg_hidden_validator_seats_per_shard.reduce((a, b) => a + b);
     return result;
 }
 
 async function showValidatorsTable(near, blockNumberOrHash) {
     const result = await validatorsInfo(near, blockNumberOrHash);
-    const seatPrice = validators.findSeatPrice(result.current_validators, result.numSeats);
+    const seatPrice = validators.findSeatPrice(
+        result.current_validators,
+        result.numSeats,
+        result.genesisConfig.minimum_stake_ratio[1],
```

Comment:
Well, it is better to pass the actual ratio there, so if there is [2, 5], you would get 2/5, instead of 1/5. I feel we should fix it on near-api-js side.

Reply:
@frol created a PR: https://github.com/near/near-api-js/pull/746